pallets/withdraw-teleport/src/weights.rs

```diff
@@ -0,0 +1,85 @@
+
+//! Autogenerated weights for pallet_withdraw_teleport
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-09-28, STEPS: `20`, REPEAT: `10`, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! WORST CASE MAP SIZE: `1000000`
+//! HOSTNAME: `Emilianos-MacBook-Pro.local`, CPU: `<UNKNOWN>`
+//! EXECUTION: , WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/trappist-node
+// benchmark
+// pallet
+// --chain=dev
+// --steps=20
+// --repeat=10
+// --pallet=pallet_withdraw_teleport
+// --extrinsic=*
+// --wasm-execution=compiled
+// --template=./templates/frame-weight-template.hbs
+// --output=./pallets/withdraw-teleport/src/weights.rs
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use core::marker::PhantomData;
+
+/// Weight functions needed for pallet_withdraw_teleport.
+pub trait WeightInfo {
+	fn withdraw_and_teleport() -> Weight;
+}
+
+/// Weights for pallet_withdraw_teleport using the Substrate node and recommended hardware.
+pub struct SubstrateWeight<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
+	/// Storage: `ParachainInfo::ParachainId` (r:1 w:0)
+	/// Proof: `ParachainInfo::ParachainId` (`max_values`: Some(1), `max_size`: Some(4), added: 499, mode: `MaxEncodedLen`)
+	/// Storage: `PolkadotXcm::SupportedVersion` (r:1 w:0)
+	/// Proof: `PolkadotXcm::SupportedVersion` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	/// Storage: `PolkadotXcm::VersionDiscoveryQueue` (r:1 w:1)
+	/// Proof: `PolkadotXcm::VersionDiscoveryQueue` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `PolkadotXcm::SafeXcmVersion` (r:1 w:0)
+	/// Proof: `PolkadotXcm::SafeXcmVersion` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainSystem::HostConfiguration` (r:1 w:0)
+	/// Proof: `ParachainSystem::HostConfiguration` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	/// Storage: `ParachainSystem::PendingUpwardMessages` (r:1 w:1)
+	/// Proof: `ParachainSystem::PendingUpwardMessages` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
+	fn withdraw_and_teleport() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `177`
+		//  Estimated: `3642`
+		// Minimum execution time: 87_000_000 picoseconds.
+		Weight::from_parts(88_000_000, 3642)
```

Comment:
for some reason the ref_time is different from the benchmark result:

https://github.com/paritytech/trappist/pull/296/files#diff-220649b2a41e8072e94f45ddbe66886b3fbea765f3546d9aef12367a0681461cR50

Even if this will not be final results, because they have to be properly benchmarked with the bot i'm curious about this.

Reply:
Because they are two different runs. 

This one is only used at [mock](https://github.com/paritytech/trappist/blob/e76a63d2ae4bb72628bd0d6674955eee9c55b8e3/pallets/withdraw-teleport/src/mock.rs#L244) Runtime. 

The one that will be used on the runtime itself is the one of the bot benchmark and I had to also run it locally for the [trait](https://github.com/paritytech/trappist/blob/e76a63d2ae4bb72628bd0d6674955eee9c55b8e3/runtime/trappist/src/lib.rs#L654) to be generated not the value itself as it will be overwritten.